src/styles/blocks/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+  margin: 0;
+  justify-content: space-between;
+
+  &__moyo_logo {
+    display: block;
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
+  height: 60px;
+  display: flex;
+  justify-content: flex-end;
+  padding-left: 0;
+
+  &__list {
+    display: flex;
+    margin: 0;
+    padding-left: 0;
+  }
+
+  &__item {
+    display: flex;
+    margin-left: 20px;
+  }
+
+  &__link {
+    text-align: center;
+    text-transform: uppercase;
+    color: #060b35;
+    text-decoration: none;
+    height: 60px;
+    line-height: 60px;
+  }
+
+  &__link:hover {
+    color: $blue-color;
```

Comment:
The variable `$blue-color` is used here but is not defined in this file. Ensure that it is defined in a global variables file or within this file to avoid errors.